go.mod

```diff
@@ -107,3 +107,19 @@ require (
 	golang.org/x/text v0.13.0 // indirect
 	golang.org/x/tools v0.12.0 // indirect
 )
+
```

Comment:
Please do not change the go mod file. It should be changed only in the building processing.